### PR TITLE
ADEN-5019 Track Porvata content type and vast id

### PR DESCRIPTION
**Pull request by fraszczakszymon**

### CODE REVIEW

File: extensions/wikia/AdEngine/js/lookup/prebid/priceParsingHelper.js [hunk 1]
```diff
@@ -169,10 +178,17 @@ define('ext.wikia.adEngine.lookup.prebid.priceParsingHelper', [
...
+			vastInfo;
 
 		if (vastRequest.responseXML) {
-			result = analyzeResponse(vastRequest.responseXML);
+			vastInfo = getVastId(vastRequest.responseXML);
+			result = analyzeResponse(vastInfo);
+			result.vastId = [
+				vastInfo.adSystem,
+				vastInfo.id
+			].join(':');
```
Comment (v3rth) [inline extensions/wikia/AdEngine/js/lookup/prebid/priceParsingHelper.js:190]: :heart:

File: extensions/wikia/AdEngine/js/video/player/porvata/porvataTracker.js [hunk 2]
```diff
@@ -23,6 +23,18 @@ define('ext.wikia.adEngine.video.player.porvata.porvataTracker', [
 			'wikiaAdStop': 'closed'
 		};
 
+	function getContentType(player) {
+		if (!player) {
+			return;
+		}
+
+		var ad = player.ima.getAdsManager().getCurrentAd();
```
Comment (v3rth) [inline extensions/wikia/AdEngine/js/video/player/porvata/porvataTracker.js]: ```var``` declaration on the top? :trollface:

File: extensions/wikia/AdEngine/js/video/player/porvata/porvataTracker.js [hunk 3]
```diff
@@ -23,6 +23,18 @@ define('ext.wikia.adEngine.video.player.porvata.porvataTracker', [
...
+
+		if (ad) {
+			return ad.getContentType();
+		}
+	}
```
Comment (v3rth) [inline extensions/wikia/AdEngine/js/video/player/porvata/porvataTracker.js]: Maybe try to reduce the number of `return` for this function (if it's possible). I think it shouldn't affect to the execution time

File: extensions/wikia/AdEngine/js/lookup/prebid/priceParsingHelper.js [hunk 4]
```diff
@@ -169,10 +178,17 @@ define('ext.wikia.adEngine.lookup.prebid.priceParsingHelper', [
 	 * @returns {VelesParams}
 	 */
 	function analyze(vastRequest) {
-		var result = invalidResult;
+		var result = invalidResult,
+			vastInfo;
 
 		if (vastRequest.responseXML) {
-			result = analyzeResponse(vastRequest.responseXML);
+			vastInfo = getVastId(vastRequest.responseXML);
```
Comment (dianafa) [inline extensions/wikia/AdEngine/js/lookup/prebid/priceParsingHelper.js]: nitpick: alph order

**Review by v3rth — APPROVED** [review]

**Pull request merged**